Test/ServerTelemetryChannel.Test/Shared.Tests/Helpers/Utils.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Microsoft.ApplicationInsights.WindowsServer.Channel.Helpers
+{
+    using System;
+
+    public static class Utils
+    {
+        internal static bool EqualsWithPrecision(this double value1, double value2, double precision)
```

Comment:
nit - We can use XUnit.Assert.Equal() overload that takes in a precision instead.